internal/super/flix.go

```diff
@@ -177,14 +203,20 @@ func packageCmd(
 	if err != nil {
 		return nil, err
 	}
-
 	isLocal := false
 	if getType(flixQuery) == flixPath {
 		isLocal = true
+		if flags.Save != "" {
+			// resolve template file location to relative path to be used by binding file
+			flixQuery, err = GetRelativePath(flixQuery, flags.Save)
```

Comment:
Also, this function is using `path/filepath` so all outputted paths will be in the format of the OS.

That means, on windows paths will be backslash separated.  At some point, `filepath.ToSlash` should be called on this path so that it is normalized for the OS.  Since it tends to be best to apply this transformation right before importing/exporting a path I'd say it probably belongs in flixkit itself here https://github.com/onflow/flixkit-go/blob/0e81dc829b9c8610ba6ddc6b4df55cdd84b2ece5/bindings/fcl-js.go#L47-L68

Reply:
filepath.Rel ends with calling `Clean`, which I didn't know. so the slashes will be correct depending on the OS the user is using. I see your point about flixkit being relative path aware, that would end up pushing IO logic into flixkit. I wanted it to only create the binding file and not have a IO dependency (interacting with the file system). 

https://pkg.go.dev/path/filepath#Clean
```
On Windows, Clean does not modify the volume name other than to replace occurrences of "/" with `\`. For example, Clean("//host/share/../x") returns `\\host\share\x`.
```

Great point! I don't think any changes are needed.

Reply:
I don't think there's any IO in the path module, just string operations.  The point I'm trying to make is that we need to make sure the path that ends up in the bindings is in Unix format.

i.e.
```
import "some\template\here.json"
```

Won't be valid javascript.  But beyond that even if the backslashes were escaped properly it templates generated by Windows OS wouldn't be compatible with Unix systems when they share it with other developers using Unix-based OS.  If it's normalized to Unix format it's supported by Windows & Unix, but not vice versa

Reply:
gotcha, that is a good point. I'll make that change

Reply:
Updated PR